librpc/Rpc.cpp

```diff
@@ -858,9 +862,8 @@ Json::Value Rpc::call(int _groupID, const Json::Value& request)
                 RPCExceptionType::BlockNumberT, RPCMsg[RPCExceptionType::BlockNumberT]));
 
         TransactionSkeleton txSkeleton = toTransactionSkeleton(request);
-        Transaction tx(txSkeleton.value, dev::config::SystemConfigMgr::maxTransactionGasLimit,
-            dev::config::SystemConfigMgr::maxTransactionGasLimit, txSkeleton.to, txSkeleton.data,
-            txSkeleton.nonce);
+        Transaction tx(txSkeleton.value, gasPrice, maxTransactionGasLimit, txSkeleton.to,
```

Comment:
the maxTransactionGasLimit should be obtained from the system table.

Reply:
In rpc this param is invalid, it will be overwrite in executive.
https://github.com/FISCO-BCOS/FISCO-BCOS/blob/fad8a6e25627e152e739e29b120c8b01fa7b896f/libexecutive/Executive.cpp#L111